### DISCUSSION TITLE
BlueSCSI on DEC MicroVAX 2000

### DISCUSSION THREAD

**Discussion by Neoncluster** [erichelgeson/BlueSCSI-v1#265] [General]
The DEC MicroVAX 2000 does not support SCSI HD's by default (but does support a SCSI tape drive!), so I am doing some experimental playing around to get the BlueSCSI working in this system. To my knowledge no one has tried the BlueSCSI on the MV2000 (however other microVAX systems are know to work).
At the very least there are some patches that need to be done to the system and also some updated ROMS installed. 
I will update the ROMs, and I have a friend who is making a DD (Disk Duplicator) backup of his already patched SCSI drive on his MV2000 system, and he will send me a .img file to use in my BlueSCSI v1.
So my question is: To potentially get the DD Image file working on my BlueSCSI do I just need to change the .img suffix to .hda, and rename the rest of the file name accordingly, or does the .img file need to be converted somehow?
Thanks for your time.

**Comment by erichelgeson** [comment]
File suffixes are ignored - BlueSCSI just supports "raw" images - eg the same bits that are on a real drive should be the same as on the SD card. 

Note you're on the BlueSCSI v1 Discussions - v1 does not have any code to support tape drives, though v2 does. https://github.com/BlueSCSI/blueSCSI-v2/

**Reply by Neoncluster** [reply]
Thanks for the reply Eric. Very useful. I should have been a bit more clear with my explanation. Basically the patches to the MV2000 operating system, plus the addition of the patched ROMs allow the onboard SCSI to communicate with SCSI hard drives. Without these changes only SCSI tape access is possible. I will swap out the ROMs for patched ones, and the drive that is being imaged already has the system patches applied. I’m hoping it is then just a simple case of renaming the disk image appropriately and putting it on an SD card and all will be good to go. Wishful thinking maybe.

To your knowledge, are there any things to worry about, or any special settings that need to be considered when making an image from an existing SCSI drive using DD. If so, I will relay this info to the person who is doing the imaging. 
Much thanks again.

**Reply by erichelgeson** [reply]
if it's a full `dd` of the entire disk that would be the most appropriate.
